packages/next-auth/src/jwt/index.ts

```diff
@@ -82,6 +85,8 @@ export async function getToken<R extends boolean = false>(
   } = params
 
   if (!req) throw new Error("Must pass `req` to JWT getToken()")
+  if (!secret)
+    throw new Error("Must pass `secret` if not set to JWT getToken()")
```

Comment:
Let's throw this instead: https://github.com/nextauthjs/next-auth/blob/main/packages/next-auth/src/core/errors.ts#L45

Reply:
Done in latest commit.